integration/spark/src/main/scala/org/apache/spark/sql/CarbonContext.scala

```diff
@@ -32,8 +34,22 @@ import org.carbondata.common.logging.LogServiceFactory
 import org.carbondata.core.util.CarbonProperties
 import org.carbondata.spark.rdd.CarbonDataFrameRDD
 
-class CarbonContext(val sc: SparkContext, val storePath: String) extends HiveContext(sc) {
+class CarbonContext(
+    val sc: SparkContext,
+    val storePath: String,
+    metaStorePath: String) extends HiveContext(sc) with Logging {
   self =>
+
+  def this (sc: SparkContext) = {
+    this (sc,
+      new File("./carbonstore").getCanonicalPath,
+      new File("./carbonmetastore").getCanonicalPath)
```

Comment:
we also can add a configuration  CarbonCommonConstants.META_STORE_LOCATION, defaulting to ./carbonmetastore.